### PR TITLE
Microsite/numbers api alternative

### PR DESCRIPTION
**Pull request by ramytanios**

### CODE REVIEW

File: docs/examples/subscriptions.md [hunk 1]
```diff
@@ -88,23 +111,29 @@ object View {
 
     useState { state =>
       div(
-        h1("Subscription"),
+        h1("Subscriptions"),
```
Comment (buntec) [inline docs/examples/subscriptions.md:114]: I would remove the heading here altogether.

File: docs/examples/subscriptions.md [hunk 2]
```diff
@@ -62,11 +74,22 @@ object Store {
         // subscribe to changes in user input and trigger debounced API calls
         store =>
           store.state
-            .map(_.number)
+            .map(_.userInput)
             .discrete
             .changes
             .debounce(1.second)
```
Comment (buntec) [inline docs/examples/subscriptions.md:80]: The `debounce` should come right before the `store.dispatch`.

File: docs/examples/data-fetching.md [hunk 3]
```diff
@@ -81,25 +98,31 @@ object View {
       div(
         h1("Data Fetch"),
```
Comment (buntec) [inline docs/examples/data-fetching.md:99]: I would remove the heading.

**Review by buntec — CHANGES_REQUESTED** [review]

**Comment by ramytanios** [comment]
updated 🚀

**Pull request merged**